pkg/liquidity-mining/contracts/admin/GaugeAdder.sol

```diff
@@ -64,23 +57,6 @@ contract GaugeAdder is IGaugeAdder, SingletonAuthentication, ReentrancyGuard {
         return _gaugeController;
     }
 
-    /**
-     * @notice Returns the gauge corresponding to a Balancer pool `pool` on Ethereum mainnet.
-     * Only returns gauges which have been added to the Gauge Controller.
-     * @dev Gauge Factories also implement a `getPoolGauge` function which maps pools to gauges which it has deployed.
-     * This function provides global information by using which gauge has been added to the Gauge Controller
-     * to represent the canonical gauge for a given pool address.
-     */
-    function getPoolGauge(IERC20 pool) public view override returns (ILiquidityGauge) {
```

Comment:
Keeping this would be pretty annoying with potentially more than one gauge per pool, and the simplicity this change brings is welcome.

Just to double check: are we completely sure none will need this information easily accessible on chain?

Reply:
This information is inaccurate as it is so we don't lose anything by removing it. If we needed an accurate registry in future we'd have to curate it manually.